frontend/appflowy_flutter/lib/plugins/database_view/widgets/row/cells/checklist_cell/checklist_cell_editor.dart

```diff
@@ -249,13 +226,9 @@ class _ChecklistItemState extends State<ChecklistItem> {
                     ),
                     hintText: LocaleKeys.grid_checklist_taskHint.tr(),
                   ),
-                  onSubmitted: (taskDescription) {
-                    context.read<ChecklistCellEditorBloc>().add(
-                          ChecklistCellEditorEvent.updateTaskName(
-                            widget.option.data,
-                            taskDescription,
-                          ),
-                        );
+                  onChanged: _debounceOnChangedText,
+                  onSubmitted: (description) {
+                    _submitUpdateTaskDescription(description);
```

Comment:
```suggestion
                  onSubmitted: _submitUpdateTaskDescription,
```

Reply:
I can't do this since there's another function call below.